src/login/LoginPage.jsx

```diff
@@ -167,6 +167,7 @@ class LoginPage extends React.Component {
     const isInstitutionAuthActive = !!secondaryProviders.length && !currentProvider;
     const isSocialAuthActive = !!providers.length && !currentProvider;
     const isEnterpriseLoginDisabled = getConfig().DISABLE_ENTERPRISE_LOGIN;
+    const ThirdPartyAuthPreloader = isSocialAuthActive || (isEnterpriseLoginDisabled && isInstitutionAuthActive);
```

Comment:
I believe `ThirdPartyAuthPreloader` is supposed to be a boolean flag. Could you please refactor it to `thirdPartyAuthPreloader`

Reply:
I agree, changed to `isThirdPartyAuthActive`